styles.css

```diff
@@ -0,0 +1,2153 @@
+  }
+
+  #name {
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 2px;
+    width: 275px;
+    height: 40px;
+    margin-left: 30px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #b3bac5;
+    padding: 0 20px;
+    outline: none;
+  }
+
+  #email {
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 2px;
+    width: 275px;
+    height: 40px;
+    margin-left: 30px;
+    padding: 0 20px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #b3bac5;
+    outline: none;
+  }
+
+  #msg {
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 2px;
+    width: 275px;
+    height: 90px;
+    margin-left: 30px;
+    padding: 20px 20px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #b3bac5;
+    outline: none;
+  }
+
+  #submit {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 0;
+    width: 150px;
+    height: 48px;
+    margin-left: 6rem;
+    border: 1px solid #6070ff;
+    border-radius: 2px;
+    background-color: #fff;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    cursor: pointer;
+  }
+
+  .client {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 8px 0;
+    width: 55px;
+    height: 32px;
+  }
+
+  .canopy-l {
+    display: none;
+  }
+
+  .fb {
+    width: 55px;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    display: block;
+    color: #344563;
+  }
+
+  #multipost {
+    width: 100%;
+    height: 66px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .frame2-l {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 22px;
+    width: 100%;
+    height: 32px;
+    margin-top: -4%;
+  }
+
+  .role1 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 8px 0;
+    width: 28%;
+    height: 32px;
+  }
+
+  .full {
+    width: 100%;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #7a869a;
+  }
+
+  #mob {
+    display: none;
+  }
+
+  .frame12 {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    width: 70%;
+    height: 320.15px;
+  }
```

Comment:
- [ ] Please ensure that the layout is respected as shown in the Figa design for the skills section here 👍🏼 

### Your Design
![image](https://user-images.githubusercontent.com/68030297/221352700-69cb78eb-6e6e-410a-8bce-48f5dfc2da39.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/221352719-6144cd6d-08c8-4bfa-a086-a002f42581a4.png)